translations/strings.yaml

```diff
@@ -541,3 +541,12 @@ inAppAuth:
   termsOfServiceAndPrivacyDisclaimer:
     value: By continuing, you agree to the <a href="terms-of-service">Terms of Service</a> and the <a href="privacy-notice">Privacy Notice</a>.
     comment:  Do not remove or translate the markup '<a href="terms-of-service">', '<a href="privacy-notice">', '</a>'.
+
+freeTrials:
+  startYourFreeTrial: 
+    value: "Sign up to start your %1-day free trial."
```

Comment:
This really doesn't work for localization: we don't support plurals, and most locales will have to adapt `day` to the number. 

Do we already know how many days of free trial we want to support? My suggestion is to put the actual number, not a variable, otherwise we should try to get rid of the number alltogether.

`Sign up to start your 7-day free trial.` or `Sign up to start your free trial.`. The former will require a new string if we change the period. For that reason, we should get rid of the number of days in at least one of the strings (maybe the title).